feat: document beta releases expectations (#5565)
expressjs/express: Security.md

Report security bugs in third-party modules to the person or team maintaining
the module.

> ## Beta releases

Reviewer: ```suggestion
## Pre-release Versions
```

Ah sorry I should have said in both.

> Alpha and Beta releases should be considered unstable and **not suitable for production use**.

Reviewer 2: ```suggestion
Alpha and Beta releases are unstable and **not suitable for production use**.
```

Vulnerabilities found in pre-releases should be reported according to the [Reporting a Bug](#reporting-a-bug) section.

> Due to the unstable nature of the branch it is not garanteed that the fix will be released in the next pre-release.

Reviewer 2: ```suggestion
Due to the unstable nature of the branch it is not guaranteed that any fixes will be released in the next pre-release.
```
